application/src/main/java/com/sanction/thunder/resources/VerificationResource.java

```diff
@@ -68,15 +65,10 @@
   public VerificationResource(UsersDao usersDao,
                               MetricRegistry metrics,
                               EmailService emailService,
-                              @Named("successHtml") String successHtml,
-                              @Named("verificationHtml") String verificationHtml,
-                              @Named("verificationText") String verificationText) {
+                              MessageOptions messageOptions) {
     this.usersDao = Objects.requireNonNull(usersDao);
     this.emailService = Objects.requireNonNull(emailService);
-
-    this.successHtml = Objects.requireNonNull(successHtml);
-    this.verificationHtml = Objects.requireNonNull(verificationHtml);
-    this.verificationText = Objects.requireNonNull(verificationText);
+    this.messageOptions = Objects.requireNonNull(messageOptions);
```

Comment:
Will this break anything if it turns out to actually be null?

Reply:
This will throw a NPE if it is null, but the reason I put that there is so we can catch if its null on startup and fail the application. It should never be null since `EmailModule` provides the `MessageOptions`, which should never be null! If it is null, we know something is wrong with the code, and our integration tests will definitely catch it.